ui/src/pages/Alert/index.tsx

```diff
@@ -35,8 +35,18 @@ const items: TabsProps['items'] = [
     label: '告警推送',
   },
 ];
+
+const getInitialActiveKey = () => {
+  const tailKey = location.hash.split('/')[location.hash.split('/').length - 1];
```

Comment:
Why not declare a const like `const hashFrags = location.hash.split('/')`? Splitting the same string twice is not so good.